bundles/c/c_lexer.moon

```diff
@@ -32,22 +32,23 @@ howl.aux.lpeg_lexer ->
 
   classdef = any {
     sequence {
-      c('keyword', word { 'enum', 'union' })
+      c 'keyword', word { 'enum', 'union' }
       ws^1
-      c('type_def', ident)
+      c 'type_def', ident
```

Comment:
Please avoid these kind of cosmetic changes, when it's not code that is actually touched by any modifications. It just makes the change unnecessarily verbose, and for instance for this block it makes things less consistent, as one occurrence of the original style is left. For these table listings I actually prefer the parentheses to be there.